Add two apps to list of Diet trackers

tvanantwerp opened the pull request:
I've added two apps that I've found extremely helpful to the list of Diet apps.

First is Cronometer, which is the best food tracker I've used. Beats the pro version of MyFitnessPal with ease (and at less cost). It shows way, way more data on micronutrients consumed than MFP. And it links with other tools like Fitbit to automatically include weight, activity, sleep, etc.

Second is Vora, which I use for fasting. I hear great things about Zero and would love to try it, but it's not available for Android. Vora has been a very useful alternative for me.

I hope these are useful contributions. Thank you for compiling such a great list!

tvanantwerp committed:
Add Cronometer to list of Diet apps

tvanantwerp committed:
Add Vora fasting tracker to list of diet apps

woop commented on README.MD:
```diff
@@ -88,7 +88,9 @@ Contributions welcome. Add links through pull requests or create an issue to sta
 - [Calorie Counter](https://www.caloriecount.com/) - Nutritional database and calorie counting app (iOS & Android).
+- [Cronometer](https://cronometer.com/) - Food, activity, and biometric tracker (iOS & Android).
 - [Zero](https://itunes.apple.com/us/app/zero-fasting-tracker/id1168348542?mt=8) - A simple fasting tracker used for intermittent, circadian rhythm, and custom fasting (iOS). 
+- [Vora](https://play.google.com/store/apps/details?id=com.fastapp&hl=en_US) Fasting tracker (Android).
```
Don't we have a better link than the app store? Also, need a hyphen.

tvanantwerp replied:
Good suggestion. Added missing hyphen and found a better URL. I also found a similar cross-platform landing page for Zero just now, so I've included that update in this PR too.

tvanantwerp committed:
Fixes to Vora entry

- Add missing hyphen between name/link and description
- Add link to correct landing page
- Note iOS availability

tvanantwerp committed:
Update Zero fasting app entry

- Update URL to landing page instead of app store link
- Note Android availability
